blueprints/cap/docker-compose.yml

```
@@ -0,0 +1,74 @@
+version: "3.8"
+
+services:
+  cap-web:
+    image: ghcr.io/capsoftware/cap-web:latest
+    container_name: cap-web
+    restart: unless-stopped
+    ports:
+      - "${PORT}:3000"
```

Don't use this way, instead use this way:

this avoid to expose the port to the host and remains internally

I see alot of this code across a couple templates

```suggestion
    ports:
      - "3000"
```